vms/platformvm/block/builder/builder.go

```diff
@@ -368,12 +396,14 @@ func buildBlock(
 }
 
 func packDurangoBlockTxs(
+	ctx context.Context,
 	parentID ids.ID,
 	parentState state.Chain,
 	mempool mempool.Mempool,
 	backend *txexecutor.Backend,
 	manager blockexecutor.Manager,
 	timestamp time.Time,
+	pChainHeight uint64,
```

Comment:
For my own understanding, why are we adding the `pChainHeight` for packing Durango blocks when P-Chain Durango blocks aren't allowed to include ICM messages?